src/main/java/SYSC4806/Repository/CourseRepository.java

```diff
@@ -34,6 +34,12 @@
      */
     List<Course> findByYear(@Param("year") int year);
 
+    /**
+     * Find all course names that are offered in a given program and year
+     * @param p_id The program id that the course should be chosen from
+     * @param year  The year to search through
+     * @return      A list of all course names offered in the requested year and program
+     */
     @Query(value = "SELECT c.name FROM COURSE as c INNER JOIN PROGRAM_COURSES as p_c ON c.id = p_c.courses_id WHERE p_c.programs_id = ?1 AND c.year = ?2", nativeQuery = true)
```

Comment:
Just a stylistic request, I don't know if everyone agrees (or cares at all honestly), but is it possible to separate this as multiple lines? I'm just picky and like my things formatted :)))). Would make me a happy boi.

```sql
@Query(value = "SELECT c.name FROM Course as c " +
               "INNER JOIN Program_Courses as p_c " +
               "WHERE p_c.programs_id = ?1 AND " +
               "c.year = ?2 ", nativeQuery = true)
```

Reply:
I tried to and for some reason, it broke whenever I did. I tried many combos as well :'(

Reply:
Sad reacts only. It's ok then :(